api/app/Actions/Event/AddEventAction.php

```diff
@@ -28,5 +30,9 @@ public function execute(AddEventRequest $request): void
         $event->timezone = $request->getTimezone();
 
         $this->eventRepository->save($event);
+
+        $event->eventType->owner->notify(new EventCreatedNotification($event));
+
+        event(new EventCreated());
```

Comment:
The better way is to fire the event, and then add a listener for it that will send the notification.

Reply:
@aLLeXUs Firstly call the event, then depends on it, notify the owner?